docs_zh-CN/1_exist_data_model.md

```diff
@@ -1 +1,541 @@
 # 1: 使用已有模型在标准数据集上进行推理
+MMDetection 在 [Model Zoo](https://mmdetection.readthedocs.io/en/latest/model_zoo.html) 中提供了数以百计现有的检测模型， 并支持多种标准数据集，包括 Pascal VOC，COCO，Cityscapes，LVIS 等。这份文档将会讲述如何使用这些模型和标准数据集来运行一些常见的任务，包括：
+
+- 使用现有模型在给定图片上进行推理。
+- 在标准数据集上测试现有模型。
+- 在标准数据集上训练预定义的模型。
+
+## 使用现有模型进行推理
+
+推理是指使用训练好的模型来检测图像上的目标。在 MMDetection 中，一个模型被定义为一个配置文件和对应的存储在 checkpoint 文件内的模型参数的集合。
+
+首先，我们建议从 [Faster RCNN](https://github.com/open-mmlab/mmdetection/tree/master/configs/faster_rcnn) 开始，其 [配置](https://github.com/open-mmlab/mmdetection/blob/master/configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py) 文件和 [checkpoint](http://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth) 文件在此。
+我们建议将 checkpoint 文件下载到 `checkpoints` 文件夹内。
+
+### 推理的高层编程接口
+
+MMDetection 为在图片上推理提供了 Python 的高层编程接口。下面是建立模型和在图像或视频上进行推理的例子。
+
+```python
+from mmdet.apis import init_detector, inference_detector
+import mmcv
+
+# Specify the path to model config and checkpoint file
+config_file = 'configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py'
+checkpoint_file = 'checkpoints/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth'
+
+# build the model from a config file and a checkpoint file
+model = init_detector(config_file, checkpoint_file, device='cuda:0')
+
+# test a single image and show the results
+img = 'test.jpg'  # or img = mmcv.imread(img), which will only load it once
+result = inference_detector(model, img)
+# visualize the results in a new window
+model.show_result(img, result)
+# or save the visualization results to image files
+model.show_result(img, result, out_file='result.jpg')
+
+# test a video and show the results
+video = mmcv.VideoReader('video.mp4')
+for frame in video:
+    result = inference_detector(model, frame)
+    model.show_result(frame, result, wait_time=1)
+```
+
+于 jupyter notebook 上的演示样例在 [demo/inference_demo.ipynb](https://github.com/open-mmlab/mmdetection/blob/master/demo/inference_demo.ipynb) 。
+
+注意：`inference_detector` 目前仅支持单张图片的推理。
+
+### 异步接口-支持 Python 3.7+
+对于 Python 3.7+，MMDetection 也支持异步接口。利用 CUDA 流，绑定 GPU 的推理代码不会阻塞 CPU，CPU/GPU 在单线程应用中能达到更高的利用率。无论是在不同的输入数据样本还是在一些推理流程的不同模型间，推理都被能并发地执行。
+
+您可以参考 `tests/async_benchmark.py` 来对比同步接口和异步接口的运行速度。
```

Comment:
You may want to add some descriptions, because asynchronous io has been integrated into image_demo.py.